rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -169,9 +178,12 @@ func (r *Layer1Relayer) ProcessGasPriceOracle() {
 				log.Error("UpdateGasOracleStatusAndOracleTxHash failed", "block.Hash", block.Hash, "block.Height", block.Number, "err", err)
 				return
 			}
-			r.lastGasPrice = baseFee
-			r.metrics.rollupL1RelayerLastGasPrice.Set(float64(r.lastGasPrice))
-			log.Info("Update l1 base fee", "txHash", hash.String(), "baseFee", baseFee, "isBernoulli", isBernoulli)
+
+			r.lastBaseFee = baseFee
+			r.lastBlobBaseFee = blobBaseFee
+			r.metrics.rollupL1RelayerLatestBaseFee.Set(float64(r.lastBaseFee))
+			r.metrics.rollupL1RelayerLastestBlobBaseFee.Set(float64(r.lastBlobBaseFee))
```

Comment:
```suggestion
			r.metrics.rollupL1RelayerLatestBlobBaseFee.Set(float64(r.lastBlobBaseFee))
```

Reply:
oh I missed this, fixed in [43ce845](https://github.com/scroll-tech/scroll/pull/1344/commits/43ce84515400a7dbc2ae2c01a6fd71fe961d8dce).